azext_edge/edge/providers/check/base/node.py

```diff
@@ -37,7 +38,9 @@ def check_nodes(
     check_manager = CheckManager(check_name="evalClusterNodes", check_desc="Evaluate cluster nodes")
     padding = (0, 0, 0, 8)
     target = "cluster/nodes"
-    check_manager.add_target(target_name=target, conditions=["len(cluster/nodes)>=1"])
+    check_manager.add_target(
```

Comment:
Confirming this runs with `init` `--check-cluster`

Reply:
Correct:
<img width="1019" height="461" alt="image" src="https://github.com/user-attachments/assets/a0ac7329-5b92-421e-a916-e1dac4c43fb7" />

Our AKS clusters also don't have enough memory🙃, but the linux check I hard-coded to fail to show the error